extract/extract/extractors/zalando.py

```diff
@@ -131,22 +131,18 @@ def __get_sustainability_info(
         beautiful_soup (BeautifulSoup): Parsed HTML
         title_attr (str): HTML attr of title
         description_attr (str): HTML attr of description
-        headline (str, optional): Headline on webpage. Defaults to
-            "Dieser Artikel erfüllt die folgenden Nachhaltigkeits-Kriterien:".
+        certificate_attr (str, optional): HTML attr of certificate. Defaults to
+            "cluster-certificates".
 
     Returns:
         Dict[str, str]: `dict` with name and description of found sustainability information
     """
 
     # this area is hidden by default, so we need to find the right one
-    hidden_areas = [
-        div
-        for div in beautiful_soup.find_all("div", attrs={"style": "max-height:0"})
-        if headline in div.text
-    ]
+    hidden_area = beautiful_soup.find("div", attrs={"data-testid": certificate_attr})
 
-    if hidden_areas:
-        sustainability_info_parsed = hidden_areas[0]
+    if hidden_area:
+        sustainability_info_parsed = hidden_area
```

Comment:
I think the following would be better because more explicit and less indirections

```python
if sustainability_info_parsed := beautiful_soup.find("div", attrs={"data-testid": certificate_attr}):
  ...
else:
  return {}
```